Test failure with pytest 5+

```
============================= test session starts ==============================
platform linux -- Python 3.7.4, pytest-5.1.0, py-1.8.0, pluggy-0.12.0
rootdir: /build/bleach-3.1.0
collected 311 items

tests/test_callbacks.py ..........                                       [  3%]
tests/test_clean.py .F.................................................. [ 19%]
........................................................................ [ 43%]
......................                                                   [ 50%]
tests/test_css.py ........................                               [ 57%]
tests/test_html5lib_shim.py ...................                          [ 63%]
tests/test_linkify.py .................................................. [ 80%]
..................................................                       [ 96%]
tests/test_unicode.py .........                                          [ 99%]
tests/test_utils.py ...                                                  [100%]

=================================== FAILURES ===================================
__________________________ test_only_text_is_cleaned ___________________________

    def test_only_text_is_cleaned():
        some_text = 'text'
        some_type = int
        no_type = None
    
        assert clean(some_text) == some_text
    
        with pytest.raises(TypeError) as e:
            clean(some_type)
>       assert "argument cannot be of 'type' type" in str(e)
E       assert "argument cannot be of 'type' type" in '<ExceptionInfo TypeError tblen=3>'
E        +  where '<ExceptionInfo TypeError tblen=3>' = str(<ExceptionInfo TypeError tblen=3>)

tests/test_clean.py:25: AssertionError
=============================== warnings summary ===============================
```

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,7 +1,7 @@
 -e .
 
 # Requirements to run the test suite
-pytest
+pytest<5
 pytest-wholenodeid
 flake8
 tox

